client/engine/messageservice/p2p-message-service/service.go

```diff
@@ -68,7 +68,18 @@ func (ms *P2PMessageService) AddPeers(peers []PeerInfo) {
 }
 
 // NewMessageService returns a running P2PMessageService listening on the given ip, port and message key.
-func NewMessageService(ip string, port int, me types.Address, messageKey p2pcrypto.PrivKey) *P2PMessageService {
+func NewMessageService(ip string, port int, me types.Address, pk []byte) *P2PMessageService {
+	h := &P2PMessageService{
```

Comment:
What is the significance of `h` as the variable name?

Reply:
That's a good question! I just rearranged the existing code that was using `h` but I think we should switch it to something that makes more sense.

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/1201/commits/055f5a882ac41d6da455ec9ac9940660a9c8361b